.changeset/silver-insects-change.md

```diff
@@ -0,0 +1,11 @@
+---
+'@leafygreen-ui/search-input': minor
+'@leafygreen-ui/guide-cue': minor
+'@leafygreen-ui/combobox': minor
+'@leafygreen-ui/tooltip': minor
+'@leafygreen-ui/select': minor
+'@leafygreen-ui/menu': minor
```

Comment:
I think we're missing:
- `data-picker`
- `number-input`
- `split-button`

Reply:
ahh the way I was grepping led me to miss these ones. thanks for calling them out!